Source/Model/Transaction.swift

```diff
@@ -182,7 +182,7 @@ public class Transaction: BaseModel, Mappable {
 
       - returns: A promise with the transaction.
     */
-    public func commit(otp: String) -> Promise<Transaction> {
+    open func commit(_ otp: String) -> Promise<Transaction> {
```

Comment:
Remove the `_` and add the label to the `commit`. Do the same for the other methods.